spec/smile-identity-core/web_api_spec.rb

```diff
@@ -1,17 +1,16 @@
 # frozen_string_literal: true
 
 RSpec.describe SmileIdentityCore::WebApi do
-  let(:partner_id) { '001' }
+  let(:partner_id) { ENV.fetch('SMILE_PARTNER_ID') }
   let(:default_callback) { 'www.default_callback.com' }
-  let(:api_key) { Base64.encode64(OpenSSL::PKey::RSA.new(1024).public_key.to_pem) }
-  let(:sid_server) { SmileIdentityCore::ENV::TEST }
-
+  let(:api_key) { ENV.fetch('SMILE_API_KEY') }
+  let(:sid_server) { ENV.fetch('SMILE_SERVER_ENVIRONMENT', SmileIdentityCore::ENV::TEST) }
```

Comment:
can we have a `.env.sample` file for the development flow here, with a call-out to it in the README.

Reply:
Yes, noted